messaging-experiments.json

```diff
@@ -412,5 +412,237 @@
         }
       ]
     }
+  },
+  {
+    "id": "PROTECTION-RETENTION-EXPERIMENT-RELEASE",
+    "enabled": true,
+    "filter_expression": "(env.channel == 'nightly' || env.channel == 'default')",
+    "targeting": "localeLanguageCode == 'en' && [userId, \"PROTECTION-RETENTION-EXPERIMENT-RELEASE\"]|stableSample(0.1)",
```

Comment:
Just a note that we will also need to update the sample size when launching this.